alibi_detect/od/tests/test_sr.py

```diff
@@ -7,7 +7,9 @@
 # create normal time series and one with perturbations
 t = np.linspace(0, 0.5, 1000)
 X = np.sin(40 * 2 * np.pi * t) + 0.5 * np.sin(90 * 2 * np.pi * t)
-idx_pert = np.random.randint(0, 1000, 10)
+
+np.random.seed(0)
```

Comment:
@RobertSamoilescu this is probably why the issue with `test_datasets` came up. `np.random.seed(0)` is set here instead of within `def test_sr()`, hence it's probably setting the seed globally for all tests, instead of just this specific test. This is a more general issue which I've flagged in https://github.com/SeldonIO/alibi-detect/issues/250.